src/MongoDB.Driver/MongoDB.Driver.csproj

```diff
@@ -50,6 +50,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\..\libmongocrypt_v2\bindings\cs\MongoDB.Crypt\MongoDB.Crypt.csproj" />
```

Comment:
Reference to libmongocrypt wrapper. 
Clone this repo: https://github.com/DmitryLukyanov/libmongocrypt.git and switch to this branch: `provide_logging_on_driver_side`

Reply:
2. Build solution. For example via `compile.sh`
3. Add `libmongocrypt.dll` with enabled trace here: `\cmake-build\Debug\mongocrypt.dll`

Reply:
4. Add the path where you cloned `libmongocrypt` to the line `53`